src/main/features/response/views/eligibility/send-your-response-by-email.njk

```diff
@@ -3,97 +3,109 @@
+          <li>{{ t('Download both the admission and defence forms') }}</li>
+        {% endif %}
 
-          {% if draft.response.type === ResponseType.PART_ADMISSION %}
-            {{ t('Complete both ' + externalLink('admission form N9A', asset_paths["pdf"] + '/OCON9A.pdf')+ ' and '
-               + externalLink('defence and counterclaim N9B', asset_paths["pdf"] + '/OCON9B.pdf') + ' using:') | safe }}
+        {% if draft.response.type === ResponseType.DEFENCE %}
+          {% if draft.rejectAllOfClaim.option === RejectAllOfClaimOption.COUNTER_CLAIM %}
+            <li>{{ t('Download the defence and counterclaim form') }}</li>
+          {% elseif draft.howMuchPaidClaimant.option === HowMuchPaidClaimantOption.LESS_THAN_AMOUNT_CLAIMED %}
+            <li>{{ t('Download both the admission and defence forms') }}</li>
           {% endif %}
+        {% endif %}
 
-          {% if draft.response.type === ResponseType.DEFENCE %}
-            {% if draft.rejectAllOfClaim.option === RejectAllOfClaimOption.COUNTER_CLAIM %}
-              {{ t('Complete both ' + externalLink('defence and counterclaim form N9B', asset_paths["pdf"] + '/OCON9B.pdf') + ' using:',
-              { ocon9B: [asset_paths["pdf"] + '/OCON9B.pdf']}) | safe }}
-            {% elseif draft.howMuchPaidClaimant.option === HowMuchPaidClaimantOption.LESS_THAN_AMOUNT_CLAIMED %}
-              {{ t('Complete both ' + externalLink('admission form N9A', asset_paths["pdf"] + '/OCON9A.pdf')+ ' and '
-              + externalLink('defence and counterclaim N9B', asset_paths["pdf"] + '/OCON9B.pdf') + ' using:') | safe }}
-            {% endif %}
-          {% endif %}
+        <li>{{ t('Complete the form on your computer using:') }}</li>
 
-          <ul class="list list-bullet">
+        <ul class="list list-bullet">
               <li>{{ t('Claim No. {{ claimNumber }}', { claimNumber: claim.claimNumber }) }}</li>
               <li>{{ t('Claimant {{ claimantName }}', { claimantName: claim.claimData.claimant.name}) }}</li>
               <li>{{ t('Defendant {{ defendantName }}', { defendantName: claim.claimData.defendant.name}) }}</li>
           </ul>
-        </li>
-        <li>{{ t('Save the completed form to your device') }}</li>
+
+        <li>{{ t('Save the completed form to your computer') }}</li>
         <li>
-        {{ t('Email the forms to '
+        {{ t('Email the form to '
         +'<a href="mailto:{{ supportEmailAddress }}?Subject={{ claimNumber }}">{{ supportEmailAddress }}</a>"',
         { supportEmailAddress: supportEmailAddress , claimNumber: claim.claimNumber }) | safe }}
         </li>
       </ul>
 
-      {{ supportWidget() }}
-      {% include "includes/service_survey_link.njk" %}
+      {% if draft.response.type === ResponseType.DEFENCE %}
+        {% if draft.rejectAllOfClaim.option === RejectAllOfClaimOption.COUNTER_CLAIM %}
 
-    </div>
+          <h2 class="heading-medium">{{ t('Counterclaim fee') }}</h2>
```

Comment:
Please use the feesTable macro I've created: 
 {{ feesTable('View claim fees', rows ) }}